generator/internal/dart/annotate.go

```diff
@@ -121,22 +122,40 @@ type packageDependency struct {
 	Constraint string
 }
 
+type AnnotateModel struct {
```

Comment:
This can be private to the `dart` package I think?

```suggestion
type annotateModel struct {
```

https://go.dev/doc/effective_go#names